blockball-bukkit-plugin/src/main/java/com/github/shynixn/blockball/bukkit/logic/business/service/BallEntityServiceImpl.kt

```diff
@@ -67,51 +76,72 @@ class BallEntityServiceImpl @Inject constructor(private val pluginProxy: PluginP
         return ballProxy
     }
 
+    /**
+     * Registers entities on the server when not already registered.
+     * Returns true if registered. Returns false when not registered.
+     */
+    override fun registerEntitiesOnServer(): Boolean {
+        if (registered) {
+            return false
+        }
+
+        val slimeClazz = findClazz("com.github.shynixn.blockball.bukkit.logic.business.nms.VERSION.BallHitBox", pluginProxy)
+        val armorStandClazz = findClazz("com.github.shynixn.blockball.bukkit.logic.business.nms.VERSION.BallDesign", pluginProxy)
+
+        entityRegistry.register(slimeClazz, EntityType.SLIME)
+        entityRegistry.register(armorStandClazz, EntityType.ARMORSTAND)
```

Comment:
Do not register the Armorstand! This is very dangerous and can cause chunk corruption. 

I know that PetBlocks contains the EntityType.Armorstand field in the enum, however it is unused.

Reply:
I couldn't reckon that armorstand should never get registered. Okay